specification/batch/data-plane/readme.md

```diff
@@ -305,7 +305,7 @@ python:
 ``` yaml $(python) && $(python-mode) == 'update'
 python:
   no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/azure-batch/azure/batch
+  output-folder: $(python-sdks-folder)/sdk/azure-batch/azure/batch
```

Comment:
Should be
```
output-folder: $(python-sdks-folder)/batch/azure-batch/azure/batch
```
https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/batch